tensorflow/lite/micro/micro_profiler.cc

```diff
@@ -67,4 +67,65 @@ void MicroProfiler::LogCsv() const {
 #endif
 }
 
+void MicroProfiler::LogTicksPerTagCsv() {
+#if !defined(TF_LITE_STRIP_ERROR_STRINGS)
+  MicroPrintf(
+      "\"Unique Tag\",\"Total ticks across all events with that tag.\"");
+  int total_ticks = 0;
+  for (int i = 0; i < num_events_; ++i) {
+    uint32_t ticks = end_ticks_[i] - start_ticks_[i];
+    int position = FindExistingOrNextPosition(tags_[i]);
+    total_ticks_per_tag[position].tag = tags_[i];
+    total_ticks_per_tag[position].ticks =
+        total_ticks_per_tag[position].ticks + ticks;
+    total_ticks += ticks;
+  }
+
+  for (int i = 0; i < num_events_; ++i) {
+    TicksPerTag each_tag_entry = total_ticks_per_tag[i];
+    if (each_tag_entry.tag == nullptr) {
+      break;
+    }
+    MicroPrintf("%s,%d", each_tag_entry.tag, each_tag_entry.ticks);
+  }
+  MicroPrintf("total,%d", total_ticks);
+#endif
+}
+
+/**
```

Comment:
super nit: let's use // for the comment style for consistency. 

https://google.github.io/styleguide/cppguide.html#Comment_Style

Reply:
Thank you for the suggestion.